ros_buildfarm/release_job.py

```diff
@@ -764,17 +774,20 @@ def configure_import_package_job(
     return (job_name, job_config)
 
 
-def get_import_package_job_name(rosdistro_name):
+def get_import_package_job_name(rosdistro_name, package_format):
     view_name = get_release_job_prefix(rosdistro_name)
-    return '%s_import-package' % view_name
+    return '%s_import-package%s' % (
+        view_name, '' if package_format == 'deb' else '-' + package_format)
 
 
-def _get_import_package_job_config(build_file):
-    template_name = 'release/deb/import_package_job.xml.em'
+def _get_import_package_job_config(build_file, package_format):
+    template_name = 'release/%s/import_package_job.xml.em' % package_format
     job_data = {
         'target_queue': build_file.target_queue,
         'abi_incompatibility_assumed': build_file.abi_incompatibility_assumed,
         'notify_emails': build_file.notify_emails,
+        'ros_buildfarm_repository': get_repository(),
+        'credential_id': build_file.upload_credential_id,
```

Comment:
This is a little wonky. The credential ID is needed to talk to pulp API, but it is specified in each build file even though this job is not specific to one build file. So if build files specify different credentials, the last one will win.

Should I make the import job specific to each build file? If I do that here, I'll likely have to do something similar for the sync-to-main job, which is currently a global job.

Reply:
> if build files specify different credentials, the last one will win.

The same is already true for the other information used from the build file, e.g. `notify_emails`. So I would say you can keep it as is within this PR.

Separate jobs per build file won't work easily since only one of the jobs (per repo type) can run in parallel.